lib/i14y_collections.rb

```diff
@@ -22,7 +26,7 @@ def self.get(handle)
   end
 
   def self.search(params)
-    response = i14y_connection.get "#{API_ENDPOINT}/search", params
-    response.body
+    response = cached_connection.get "#{API_ENDPOINT}/search", params
```

Comment:
I think we decided to do parenthesis around method calls with arguments.

Reply:
While your in this file can you update line 2 to be API_ENDPOINT = "/api/v1/collections".freeze?  :)

Reply:
Good suggestions - updated.

Note: I froze strings in those files using the `# frozen_string_literal: true` magic comment, as that will be the default in Ruby 3.